pkg/config/cmd/cmd.go

```diff
@@ -35,13 +35,18 @@ type Mapper interface {
 	InfoForObject(obj runtime.Object, preferredGVKs []unversioned.GroupVersionKind) (*resource.Info, error)
 }
 
+// ErrorFilterFunc provides a way to filter errors during the Bulk.Run.  This may be used in conjunction with
+// BulkAction.WithMessageAndPrefix if you are reporting items as warnings.
+type ErrorFilterFunc func(e error) bool
```

Comment:
Not very clear to me what the return value means, does a `true` means "filter out" or "keep it"? Mention it in the doc, or call it `IgnoreErrorFunc`, `BypassErrorFunc`, `FatalErrorFunc`, or something similar.